nomos-services/storage/src/lib.rs

```diff
@@ -144,28 +150,35 @@ impl<Backend: StorageBackend> Debug for StorageMsg<Backend> {
                 write!(f, "Remove {{ {key:?} }}")
             }
             Self::Execute { .. } => write!(f, "Execute transaction"),
+            Self::Api { .. } => {
+                write!(f, "Api {{ .. }}")
+            }
         }
     }
 }
 
 /// Storage error
 /// Errors that may happen when performing storage operations
 #[derive(Debug, thiserror::Error)]
-enum StorageServiceError<Backend: StorageBackend> {
+pub enum StorageServiceError<Backend: StorageBackend> {
     #[error("Couldn't send a reply for operation `{operation}` with key [{key:?}]")]
     ReplyError { operation: String, key: Bytes },
     #[error("Storage backend error")]
     BackendError(#[source] Backend::Error),
 }
 
 /// Storage service that wraps a [`StorageBackend`]
-pub struct StorageService<Backend: StorageBackend + Send + Sync + 'static, RuntimeServiceId> {
+pub struct StorageService<Backend, RuntimeServiceId>
+where
+    Backend: StorageBackend + StorageBackendApi + Send + Sync + 'static,
```

Comment:
I guess `StorageBackend` bound is not needed since `StorageBackendApi` is a sub-trait.